certified-connectors/Zippydoc/apiDefinition.swagger.json

```diff
@@ -0,0 +1,258 @@
+      },
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "operationId": "OnTableChange",
+        "summary": "On table changed",
+        "description": "Triggers flow when any table is modified.",
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "x-ms-notification-url": {
+                  "type": "string",
+                  "description": "x-ms-notification-url",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal",
+                  "title": ""
+                },
+                "type": {
+                  "type": "string",
+                  "description": "type",
+                  "title": "",
+                  "enum": [
+                    "ON_TABLE_CHANGE"
+                  ]
+                }
+              },
+              "required": [
+                "x-ms-notification-url"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/power-automate/{webHookId}": {
+      "delete": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Delete WebHook",
+        "description": "Called automatically by power-automate",
+        "operationId": "DeleteWebHook",
+        "parameters": [
+          {
+            "name": "webHookId",
+            "in": "path",
+            "required": true,
+            "x-ms-url-encoding": "single",
+            "type": "string"
+          }
+        ],
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/document/power-automate/{name}": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Execute",
+        "description": "Executes all tasks in defined flow.",
+        "operationId": "Execute",
+        "parameters": [
+          {
+            "name": "name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Name of the flow that should execute all of its tasks.",
+            "x-ms-url-encoding": "single",
+            "x-ms-summary": "Name of the flow that should execute all of its tasks."
+          }
+        ]
+      }
+    },
+    "/excel/table/power-automate/upload": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
@worm333 mentioned in the comment above. Actions in your operations should not have default responses with an empty schema. Please define exact responses for your actions.